_posts/2010-03-31-agarwal10a.md

```diff
@@ -3,12 +3,10 @@ title: Optimal Allocation Strategies for the Dark Pool Problem
 abstract: We study the problem of allocating stocks to dark pools. We propose and
   analyze an optimal approach for allocations, if continuous-valued allocations are
   allowed. We also propose a modification for the case when only integer-valued allocations
-  are possible. We extend the previous work on this problem by Ganchev et al (UAI
-  2009) to adversarial scenarios, while also improving over their results in the iid
-  setup. The resulting algorithms are efficient, and are tested on extensive simulations
-  under stochastic and adversarial inputs. Our work also has consequences for other
-  perishable inventory control problems, extending their analyses to adversarial models
-  too.
+  are possible. We extend the previous work on this problem (Ganchev et al., 2009)
+  to adversarial scenarios, while also improving over their results in the iid
+  setup. The resulting algorithms are efficient, and perform well in simulations
+  under stochastic and adversarial inputs.
```

Comment:
Following the manuscript's abstract
<img width="303" alt="Screenshot 2022-02-11 at 5 03 02" src="https://user-images.githubusercontent.com/7121753/153487567-e856e194-3d40-473d-928e-a52a01928b32.png">


PDF: http://proceedings.mlr.press/v9/agarwal10a/agarwal10a.pdf